AssemblyCalc.asm

```diff
@@ -0,0 +1,86 @@
+# COMP3410
+# Author: Miguel Betances
+# Assignment: PA[1]
+# Date: Feb/11/2015
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# Calculator.asm
+# Input: two integers
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+	.data
+
+	head: .asciiz "\nMiguel Betances: Assembly Aritmetic Calculator\n"
+
+		input1: .asciiz 	"Enter first number: \n" 	#Ask user for input
+		input2: .asciiz 	"Enter second number: \n" 	#Ask user for input
+		result1: .asciiz 	"\nAddition result:  "
+		result2: .asciiz	"\nSubstraction result:  "
+		result3: .asciiz 	"\nProduct result :  "
+		done:	.asciiz 	"\nDone\n" 			#Done 
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	
+	#Display welcome
+		li $v0,4
+		la $a0, head
+		syscall
+	#Asking for user input 
+		li $v0,4
+		la $a0, input1
+		 syscall
+
+		li $v0,5	#Store the user input
+		syscall
+
+		move $t0, $v0	#Move the user input value to temporary register t0
```

Comment:
-5 Not limited to 9 fibonacci numbers.  Very dangerous to allow the user to exceed the bounds of the array set up in the beginning because you may overwrite other important addresses.  May or may not force the printing of unintended characters, depending on whether or not one of the fibonacci numbers writes over any of the print strings.